app/uk/gov/hmrc/cipphonenumberverification/services/PasscodeService.scala

```diff
@@ -34,9 +34,9 @@ class PasscodeService @Inject()(passcodeCacheRepository: PasscodeCacheRepository
       .map(_ => phoneNumberAndOtp)
   }
 
-  def retrievePasscode(phoneNumberAndOtp: PhoneNumberAndOtp): Future[Option[PhoneNumberAndOtp]] = {
-    logger.debug(s"Retrieving phoneNumberAndOtp from database for ${phoneNumberAndOtp.phoneNumber}")
-    passcodeCacheRepository.get[PhoneNumberAndOtp](phoneNumberAndOtp.phoneNumber)(DataKey("cip-phone-number-verification"))
+  def retrievePasscode(phoneNumber: String): Future[Option[PhoneNumberAndOtp]] = {
```

Comment:
same here

Reply:
Because we are now using the same function from verify/otp. verify/otp has a PhoneNumberAndOtp object so I changed the signature to take a string so that it will work for both verify and verify/otp